zivid_camera/srv/CaptureAssistantSuggestSettings.srv

```diff
@@ -0,0 +1,7 @@
+uint8 AMBIENT_LIGHT_FREQUENCY_NONE=0
+uint8 AMBIENT_LIGHT_FREQUENCY_50HZ=1
+uint8 AMBIENT_LIGHT_FREQUENCY_60HZ=2
+
+duration max_capture_time
+uint8 ambient_light_frequency
+---
```

Comment:
For me it would make sense that the user could get the suggested settings from the Capture Assistant in the service response, and then he could provide the settings to to the dynamic reconfigure server (or to `capture(camera, settings)`).

But I guess the decision to do it like this is based on that we are already using the dynamic reconfigure server for the settings? So that if the user wants to check the output from the Capture Assistant, he will query the dynamic reconfigure server and alternatively disable some of the frames suggested by the Capture Assistant there, if that is desireable?

Reply:
We chose to use dynamic_reconfigure for the settings tree. If we do as you propose, then we need a whole system for auto generating the messages to represent the settings tree, which has its own system compared to dynamic_reconfigure. We still need dynamic_reconfigure for having a way to visualize the settings in a GUI, and it is more common to use dynamic_reconfigure for this. I don't see that this gives much value, I would prefer to have one system for simplicity.

Reply:
@runenordmo As discussed IRL, frame importance is different for different applications/users. Some may prefer speed over coverage, and others may want the opposite.

Reply:
Yes, so they can just disable/enable frames in dynamic reconfigure based on their preference.